src/components/ha-filter-devices.ts

```diff
@@ -248,7 +248,7 @@ export class HaFilterDevices extends LitElement {
         }
         search-input-outlined {
           display: block;
-          padding: 0 8px;
+          padding: 4px 8px 0;
```

Comment:
Can you update these to use the new(ish) design token variables we now have in https://github.com/home-assistant/frontend/blob/221ca56121a6d8870ba359bb54ca07d797e55a1d/src/resources/theme/core.globals.ts#L23-L44

Reply:
I see, ok, will do

Reply:
Hmm, I see no feedback values are provided in a source code for these vars...